src/main/kotlin/no/nav/syfo/Queries.kt

```diff
@@ -25,11 +25,10 @@ const val queryStatusRetrieve =
     ORDER BY sykmeldt_fnr, virksomhet_nr, opprettet DESC
 """
 
-fun DatabaseInterface.addStatus(fnr: SykmeldtFnr, ident: VeilederIdent, enhetNr: EnhetNr, virksomhetNr: VirksomhetNr) {
-    val uuid = UUID.randomUUID().toString()
+fun DatabaseInterface.addStatus(uuid: UUID, fnr: SykmeldtFnr, ident: VeilederIdent, enhetNr: EnhetNr, virksomhetNr: VirksomhetNr) {
```

Comment:
Trenger `uuid` å være av typen `UUID` her, i stedet for `string`? Nå gjør vi `uuid.toString()` når vi lager objekt for kafka, så `uuid.fromString()` når vi leser fra kafka, og så `toString` igjen her? Vi kunne vel bare hatt string hele veien?

Reply:
Kan endre til det ja

Reply:
Endret.